modules/cnv-pxe-booting-with-mac-address.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-pxe-booting-with-mac-address-{context}"]
+= PXE booting with a specified MAC address
+
+PXE booting, or network booting, is supported in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
+.Prerequisites
+
+* A bridge must be configured on all nodes. The bridge must be connected to
+an interface that allows PXE booting.
+
+.Procedure
+
+. Configure a PXE network on the cluster:
+
+.. Create the NetworkAttachmentDefinition file for PXE network `pxe-net-conf`:
++
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: pxe-net-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "type": "cnv-bridge",
+      "bridge": "br1"
+    }'
+----
++
+[NOTE]
+====
+In this example, the VMI will be attached through a trunk port
```

Comment:
this is not a trunk port. the port on the host here need to be configured as access.

if you want the pod to be trunk you need to and the vlan to the network attachment definition CRD

Reply:
Ah, that was left over from the 1.4 docs. I wasn't sure if it still applied. Do you think I should just remove that note, or should I replace it with something?

Reply:
I think you need to replace it with 
`the VMI will be attached through a access port with the requested vlan to the bridge <br1>`

Reply:
@SchSeba Does this seem sufficient? "In this example, the VMI will be attached to the bridge `br1` through an access port." 

I'm not sure if we need to mention the VLAN here. Does the admin need to configure it (and the access port) themselves? VLANs are not mentioned in this file at all, so please let me know if I need to add a prerequisite or if this is something that happens "under the hood."

Reply:
@ousleyp I think we need to document it this why.

VM -> port (vlan mode) -> linux-bridge -> bond0 (host interface port connected in trunk mode) -> network switch

Reply:
The note now reads "The VMI will be attached to the bridge `br1` through an access port with the requested VLAN." @SchSeba does that make sense?

Reply:
Yes thanks!